### DISCUSSION TITLE
Expose on different port

### DISCUSSION THREAD

**Discussion by amrap030** [fastapi-mvc/fastapi-mvc#253] [Q&A] [answered]
Hello together,

how can I expose the application on a different port other than 8000?

I tried to change `bind = os.getenv("FASTAPI_BIND", "127.0.0.1:8000")` in the `gunicorn.py` file under the config folder to `bind = os.getenv("FASTAPI_BIND", "127.0.0.1:7000")` but it is still exposed on 8000.

**Comment by rszamszur** [comment, marked as answer]
Hey @amrap030 

The `gunicorn.py` is a part of the package. For development, make sure you have installed your project in editable mode to always have the latest changes. However, you can still override the `gunicorn.py` default port by one of the following:
1. Using environment variable: `export FASTAPI_BIND=127.0.0.1:7000`
2. Using CLI `serve` command `--bind` option: `$YOUR_PROJECT_SCRIPT_NAME serve --bind 127.0.0.1:7000`
